demisto_sdk/commands/content_graph/objects/content_item.py

```diff
@@ -84,16 +84,25 @@ def support_level(self) -> str:
 
     @property
     def ignored_errors(self) -> list:
+        return self.get_ignored_errors(self.path.name)
+
+    def ignored_errors_related_files(self, file_path: str) -> list:
+        return self.get_ignored_errors((Path(file_path)).name)
+
+    def get_ignored_errors(self, path: str) -> list:
```

Comment:
```suggestion
    def get_ignored_errors(self, path: Union[str, Path]) -> list:
```

any way you use it within fstring, so no reason why not to receive Path